torch/autograd/gradcheck.py

```diff
@@ -285,6 +285,80 @@ def get_numerical_jacobian_wrt_specific_input(fn, input_idx, inputs, outputs, ep
         jacobian_cols[d_idx] = _compute_numerical_jvps_wrt_specific_input(jvp_fn, eps, x.is_complex())
     return _combine_jacobian_cols(jacobian_cols, outputs, input, input.numel())
 
+def _get_analytical_jacobian_forward_ad(fn, inputs, outputs, *, all_u=None) -> Tuple[Tuple[torch.Tensor, ...], ...]:
+    """Computes the analytical Jacobian using forward mode AD of `fn(inputs)` with respect
```

Comment:
Maybe reword as: Computes the analytical Jacobian of `fn(inputs)` using forward-mode AD